README.md

```diff
@@ -84,6 +84,13 @@ Ask [@acabunoc](http://github.com/acabunoc) for ones marked `###########`. Our c
 
 To run the application successfully you need to have [mongodb](https://www.mongodb.org/) server and [redis-server](http://redis.io/download) running locally. You can install these from their offical website or use your favorite package manager.
 
+#### Adding user as a publisher
```

Comment:
I think this information might be best under [/docs](https://github.com/mozillascience/PaperBadger/tree/master/docs) instead of the README

Do you also have instructions for getting ORCID auth working locally?